sigmahq/sigmahq_conventions.md

```diff
@@ -67,7 +69,8 @@ All newly created rules must start with a status of `experimental`
 
 ## References
 
-All rules must provide a public reference when possible
+- All rules must provide a public reference when possible
+- References to git based platforms such as Github or Gitlab must provide a permalink instead of the main branch link. This is to avoid any future confusion in the intended reference in case the maintainer make changes
```

Comment:
```suggestion
- References to git-based platforms such as Github or Gitlab must provide a permalink instead of the main branch link. This is to avoid any future confusion in the intended reference in case the maintainers make changes
```